sci-libs/vtk/vtk-7.0.0.ebuild

```diff
@@ -0,0 +1,362 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+WEBAPP_OPTIONAL=yes
+WEBAPP_MANUAL_SLOT=yes
+
+inherit flag-o-matic java-pkg-opt-2 python-single-r1 qmake-utils versionator toolchain-funcs cmake-utils virtualx webapp
+
+# Short package version
+SPV="$(get_version_component_range 1-2)"
+
+DESCRIPTION="The Visualization Toolkit"
+HOMEPAGE="http://www.vtk.org/"
+SRC_URI="
+	http://www.${PN}.org/files/release/${SPV}/VTK-${PV}.tar.gz
+	doc? ( http://www.${PN}.org/files/release/${SPV}/${PN}DocHtml-${PV}.tar.gz )
+	test? (
+		http://www.${PN}.org/files/release/${SPV}/VTKData-${PV}.tar.gz
+		http://www.${PN}.org/files/release/${SPV}/VTKLargeData-${PV}.tar.gz
+		)
+	"
+
+LICENSE="BSD LGPL-2"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+SLOT="0"
+IUSE="
+	all-modules aqua boost doc examples imaging ffmpeg gdal java json kaapi mpi
+	mysql odbc offscreen postgres python qt5 rendering tbb test theora tk tcl
+	video_cards_nvidia views web xdmf2 R +X"
+
+REQUIRED_USE="
+	all-modules? ( python xdmf2 boost )
+	java? ( qt5 )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	tcl? ( rendering )
+	test? ( python )
+	tk? ( tcl )
+	web? ( python )
+	^^ ( X aqua offscreen )
+	"
+
+RDEPEND="
+	dev-libs/expat
+	dev-libs/jsoncpp
+	dev-libs/libxml2:2
+	>=media-libs/freetype-2.5.4
+	media-libs/libpng:0=
+	media-libs/mesa
+	media-libs/libtheora
+	media-libs/tiff:0
+	sci-libs/exodusii
+	sci-libs/hdf5:=
+	sci-libs/netcdf-cxx:0=
+	sys-libs/zlib
+	virtual/jpeg:0
+	virtual/opengl
+	>=x11-libs/gl2ps-1.3.8
+	x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXt
+	boost? ( dev-libs/boost:=[mpi?] )
+	examples? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		sci-libs/vtkdata
+	)
+	ffmpeg? ( virtual/ffmpeg )
+	gdal? ( sci-libs/gdal )
+	java? ( >=virtual/jre-1.5:* )
+	kaapi? ( <sci-libs/xkaapi-3 )
+	mpi? (
+		virtual/mpi[cxx,romio]
+		python? ( dev-python/mpi4py[${PYTHON_USEDEP}] )
+		mysql? ( virtual/mysql )
+		odbc? ( dev-db/unixODBC )
+		offscreen? ( media-libs/mesa[osmesa] )
+		postgres? ( dev-db/postgresql:= )
+		python? (
```

Comment:
Yes that makes sense. I will move the closing ")" according to the old indentation.